plugins/inputs/x509_cert/x509_cert.go

```diff
@@ -204,25 +220,45 @@ func getTags(cert *x509.Certificate, location string) map[string]string {
 	return tags
 }
 
+func (c *X509Cert) collectCertURLs() ([]*url.URL, error) {
+	var urls []*url.URL
+
+	for _, globpath := range c.globpaths {
+		files := globpath.Match()
+		if len(files) <= 0 {
+			c.Log.Errorf("could not find file: %v", globpath)
+			continue
+		}
+		for _, file := range files {
+			file = "file://" + file
+			u, err := url.Parse(file)
```

Comment:
Looks like this causes windows tests to fail, as it assumes the path is a linux-style path. On windows the path will start with `C:` or something and it will recognize the colon as a port. A workaround for windows is adding an extra slash to the `file://` part but it looks like there might be other issues (see: https://github.com/golang/go/issues/13276). I'm thinking maybe we'd just need to manually handle the windows case here, not 100% sure but we will need it to work on windows.